wiki/Announcements/Contests/Monthly_Fanart_Contest/A_very_osu!_halloween/en.md

```diff
@@ -0,0 +1,95 @@
+# A very osu! Halloween
+
+## Public Picks
+
+### First Place
```

Comment:
wouldn't this be redundant since I would need to put the user link anyways (links should not be in the headings btw)